pinot-common/src/main/java/org/apache/pinot/common/utils/request/RequestUtils.java

```diff
@@ -89,12 +91,13 @@ public static void setOptions(SqlNodeAndOptions sqlNodeAndOptions, JsonNode json
       }
     }
     if (jsonRequest.has(CommonConstants.Broker.Request.QUERY_OPTIONS)) {
-      Map<String, String> queryOptionsFromJson = RequestUtils.getOptionsFromJson(jsonRequest,
-          CommonConstants.Broker.Request.QUERY_OPTIONS);
+      Map<String, String> queryOptionsFromJson =
+          RequestUtils.getOptionsFromJson(jsonRequest, CommonConstants.Broker.Request.QUERY_OPTIONS);
       queryOptions.putAll(queryOptionsFromJson);
     }
-    boolean enableTrace = jsonRequest.has(CommonConstants.Broker.Request.TRACE) && jsonRequest.get(
-        CommonConstants.Broker.Request.TRACE).asBoolean();
+    boolean enableTrace =
```

Comment:
did we get a new coding style? or those files were just too old and not reformatted with newer styles.

Reply:
Reverted the unrelated reformatting